index.html

```diff
@@ -11610,9 +11610,9 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 		<div class="state" id="aria-expanded">
 			<sdef>aria-expanded</sdef>
 			<div class="state-description">
-				<p><a>Indicates</a> whether a grouping element owned or controlled by this element is expanded or collapsed.</p>
-				<p>The <sref>aria-expanded</sref> attribute is applied to a focusable, interactive element that toggles visibility of content in another element. For example, it is applied to a parent <rref>treeitem</rref> to indicate whether its child branch of the tree is shown. Similarly, it can be applied to a <rref>button</rref> that controls visibility of a section of page content.</p>
-				<p>If a grouping container that can be expanded or collapsed is not [=ARIA/owned=] by the element that has the <sref>aria-expanded</sref> attribute, the author SHOULD identify the controlling relationship by referencing the container from the element that has <sref>aria-expanded</sref> with the <pref>aria-controls</pref> property.</p>
+				<p><a>Indicates</a> whether a related grouping element is expanded or collapsed.</p>
+				<p>The <sref>aria-expanded</sref> attribute is applied to a focusable, interactive element that toggles visibility of content in a different grouping element. If the element with <sref>aria-expanded</sref> is also a <rref>treeitem</rref> in a <rref>tree</rref> or a <rref>row</rref> in a <rref>treegrid</rref>, then it SHOULD also be the <a>accessibility parent</a> of the grouping element it expands and collapses. Otherwise, the element with <sref>aria-expanded</sref> SHOULD NOT also be the <a>accessibility parent</a> of its grouping element and instead identify that relationship using <pref>aria-controls</pref>.</p>
+				<p class="note">For example, <sref>aria-expanded</sref> is applied to a parent <rref>treeitem</rref> to indicate whether its child branch of the tree is shown. Similarly, it can be applied to a <rref>button</rref> that controls visibility of a section of page content.</p>
```

Comment:
other minor wording to try and stay consistent with previous suggestions.

also, rather than a note, this seems like it'd be the leadin to a nice short example to show some simple markup to better illustrate the differences.

```suggestion
				<p class="note">For example, <sref>aria-expanded</sref> is applied to a parent <rref>treeitem</rref> to indicate whether its child branch of the tree is shown. Similarly, it can be applied to a <rref>button</rref> to control the visibility of an element and its content, on the current page.</p>
```